static/app/components/assigneeBadge.stories.tsx

```diff
@@ -39,10 +40,30 @@ export default storyBook(AssigneeBadge, story => {
     const [chevron1Toggle, setChevron1Toggle] = useState<'up' | 'down'>('down');
     const [chevron2Toggle, setChevron2Toggle] = useState<'up' | 'down'>('down');
 
+    const team: Team =
+      teams.length > 0
+        ? teams[0]
+        : {
+            id: '1',
+            slug: 'team-slug',
+            name: 'Team Name',
+            access: ['team:read'],
+            teamRole: null,
+            isMember: true,
+            memberCount: 0,
+            avatar: {avatarType: 'letter_avatar', avatarUuid: uuid4()},
+            flags: {
+              'idp:provisioned': false,
+            },
+            externalTeams: [],
+            hasAccess: false,
+            isPending: false,
```

Comment:
could probably directly use `TeamFixture` here instead of hardcoding a fake team but this also works

Reply:
I tried that but for some reason the import wasn't resolving correctly so I just copy pasted the teamfixture object